src/libostree/ostree-sign-ed25519.c

```diff
@@ -222,29 +236,27 @@ ostree_sign_ed25519_data_verify (OstreeSign *self, GBytes *data, GVariant *signa
       g_autoptr (GVariant) child = g_variant_get_child_value (signatures, i);
       g_autoptr (GBytes) signature = g_variant_get_data_as_bytes (child);
 
+#if defined(USE_LIBSODIUM)
```

Comment:
This is minor but I think things would be slightly cleaner if we did something at the top like:

```
#define FIXED_KEY_SIZES (defined(USE_LIBSODIUM))
```

And here did `#if FIXED_KEY_SIZES` or so? (Or alternatively invert with `DYNAMIC_KEY_SIZES`...no strong opinion here)